runtime/compiler/p/runtime/PicBuilder.spp

```diff
@@ -1017,6 +1023,70 @@ _interpreterUnresolvedStringGlue:
 	b	.L_patch_dword_load
 	endproc._interpreterUnresolvedStringGlue:
 
+#ifdef AIXPPC
+._interpreterUnresolvedConstantDynamicGlue:
+   .function ._interpreterUnresolvedConstantDynamicGlue,startproc._interpreterUnresolvedConstantDynamicGlue,16,0,(endproc._interpreterUnresolvedConstantDynamicGlue-startproc._interpreterUnresolvedConstantDynamicGlue)
+#elif defined(LINUXPPC64)
+FUNC_LABEL(_interpreterUnresolvedConstantDynamicGlue):
+#else
+_interpreterUnresolvedConstantDynamicGlue:
+._interpreterUnresolvedConstantDynamicGlue:
+#endif
+	startproc._interpreterUnresolvedConstantDynamicGlue:
+#include "p/runtime/SaveGPRs.inc"
+	mfcr    r0
+	stw     r0,-4(J9SP)					! Save CR also
+	addi    J9SP,J9SP,-(33*ALen)
+	mfspr	r7, LR						! Get the RA in snippet
+	laddr	RTOC, J9TR_VMThreadRTOCOffset(J9VM_STRUCT)	! Restore RTOC
+#ifdef AIXPPC
+	laddr	r8, TOCjitResolveConstantDynamic(RTOC)		! Load descriptor pointer
+	laddr	r8, 0(r8)					! Load the callee address
+#elif defined(LINUXPPC64)
+	laddr	r8, TOCjitResolveConstantDynamic@toc(RTOC)	! Load descriptor pointer
+#if !defined(__LITTLE_ENDIAN__)
+	laddr	r8, 0(r8)					! Load the callee address
+#endif
+#else
+	laddr	r8, jitResolveConstantDynamic@got(RTOC)		! Load the callee address
+#endif
+	laddr	r3, 1*ALen+4(r7)				! Load constant pool literal
+	lwz	r4, 1*ALen(r7)					! Load cp index for this field
+	laddr	r5, 0(r7)					! Load code cache RA
+	rlwinm	r4, r4, 0, 5, 31				! Masking off the flag bits
+	addi	r5, r5, 1					! Increament for EX search
+	mtspr	CTR, r8						! Move callee address to CTR
+	bcctrl	BO_ALWAYS, CR0_LT				! Call to resolve: RTOC, r7
+	laddr	r6, 0(r7)					! Load code cache RA
+	rlwinm  r17, r3, 0, 31, 31				! keep in r17 if r3 last bit is set: <clinit> is being executed
+#ifdef TR_HOST_64BIT
+	clrrdi r3, r3, 1                                        ! <clinit> clear up the last bit, which must be zero
+#else
+	rlwinm  r3, r3, 0, 0xfffffffe                           ! <clinit> clear up the last bit, which must be zero
+#endif
+	cmpi	cr0, 0, r17, in_clinit				! <clinit> is being executed
```

Comment:
Indentations look quite odd with the tabs.

Reply:
Looks like all this weird indentation crept in from code that was copied from `_interpreterUnresolvedStringGlue`. I'll go ahead and clean it up a little bit...

Reply:
I've replaced all the tabs with spaces for now. I'm still not a big fan of the indentation, but I'm trying to keep it consistent with what's used in the other PicBuilder helpers.